changelog/232.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
Sorry to be the changelog pedant, but if it turns out that this was a bug (pending Kit's response to the comment above), we should make the release note type `bug`. That will keep it grouped in the right place when the changelog is generated. 🙂

Reply:
Yeah, I don't know why I decided to go crazy with these -- that's the last time, I promise! I've stickied all the valid changelog release-note types. Sorry about that.